config/config.json

```diff
@@ -143,7 +143,8 @@
         "EnableEmailBatching": false,
         "EmailBatchingBufferSize": 256,
         "EmailBatchingInterval": 30,
-        "SkipServerCertificateVerification": false
+        "SkipServerCertificateVerification": false,
+        "EmailNotificationContentsType": "full"
```

Comment:
This defaults to "full", even in TE instances. Licensed E20 instances can change the setting to "generic" via a UI in System Console. There is no check in the email code to see if the instance is E20, so as it's currently implemented, somebody who changed this manually in the config would be able to use the feature.